src/components/Markdown/Code.tsx

```diff
@@ -0,0 +1,41 @@
+import { Code as InlineCode } from "@chakra-ui/react";
+/* import { Language } from "prism-react-renderer"; */
+import { Children, FunctionComponent, ReactNode } from "react";
+/* import { useLanguage } from "../../hooks/useLanguage"; */
+import { Code as BlockCode } from "../Code";
+
+interface CodeProps {
+  inline?: boolean;
+  children: ReactNode;
+}
+
+export const Code: FunctionComponent<CodeProps> = ({ inline, children }) => {
+  /* const [lang] = useLanguage(); */
+
+  if (inline) {
+    return (
+      <InlineCode
+        bg="gray.50"
+        border="1px solid"
+        borderColor="gray.100"
+        borderRadius="md"
+        color="blue.500"
+        px={2}
+        py={0}
+      >
+        {children}
+      </InlineCode>
+    );
+  }
+
+  const code = Children.toArray(children)
+    .reduce((accum: string, child): string => {
+      if (typeof child === "string") {
+        return `${accum}${child}`;
+      }
+      return accum;
+    }, "")
+    .trim();
+
+  return <BlockCode code={code} language={"python"} />;
```

Comment:
Do we want the useLanguage hook here? I assume this was for testing as it's commented out above.

Reply:
This was some carry-over from your implementation, but realistically - no. We should really only use it for the api reference section so it'll need to be passed in as a prop

Reply:
Actually gonna pass it in for now - no harm in having mostly correct syntax highlighting